lodash.js

```diff
@@ -13939,6 +13933,12 @@
      * //   return __p;
      * // }
      *
```

Comment:
This is the only part of this PR that I'm not stoked with.

I had to reorder these examples because changing the interpolate regex means that all the following examples don't work correctly. I understand that these are intended to be run in isolation but markdown-doctest runs them as one big example.

Another option would be to reset the value of interpolate after this example, but that would add unnecessary noise to the docs.

Usually I would just skip this example set but because we're compiled from jsdoc I can't do that in the normal way (prepending the example with `<!-- skip-example -->` in the markdown).

If you have any ideas I would love to hear them 😄